ratelimit_test.go

```diff
@@ -7,11 +7,22 @@ import (
 
 	"go.uber.org/atomic"
 
-	"github.com/benbjohnson/clock"
 	"github.com/stretchr/testify/assert"
 )
 
+const advanceBackoffDuration = 5 * time.Millisecond
+
+func (tt *testTime) Now() time.Time {
+	return tt.now()
+}
+
+func (tt *testTime) Sleep(duration time.Duration) {
+	timer, _ := tt.newTimer(duration)
+	<-timer
+}
+
 type testRunner interface {
+	getName() string
```

Comment:
why do we need this?

The name is included in the test-name automatically via tt.run (?)

Reply:
agree